unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs

```diff
@@ -63,11 +63,11 @@ private void UpdateLODsVerticalMovementAndBillboard()
             GameObject lodGO;
             for (int i = 0; i < listCount; i++)
             {
-                lodGO = avatarsList[i].GetLODRenderer().gameObject;
+                lodGO = avatarsList[i].meshRenderer.gameObject;
                 if (!lodGO.activeSelf)
                     continue;
 
-                if (applyVerticalMovement)
+                if (applyVerticalMovement && Vector3.Distance(lodGO.transform.position, CommonScriptableObjects.playerUnityPosition.Get()) > 2 * DataStore.i.avatarsLOD.LODDistance.Get())
```

Comment:
Maybe we can move this hardcoded value `2` to some constant with a descriptive name? Or if not, at least we should put some comment above this line explaining why are we multiplying by 2 the `DataStore.i.avatarsLOD.LODDistance.Get()` value.

Reply:
Agree!

Reply:
Removed the "vertical movement" on far away impostors as nobody likes it XD, so this code doesn't exist anymore.